common/src/main/java/tds/common/configuration/RedisClusterConfiguration.java

```diff
@@ -15,11 +18,13 @@
  * able to serialize/deserialize generic Objects using the application ObjectMapper.
  */
 @Configuration
-@ConditionalOnProperty("spring.redis.sentinel.master")
+@Import(RedisAutoConfiguration.class)
+@ConditionalOnProperty(name = "tds.cache.implementation", havingValue = "redis")
```

Comment:
I had to switch to a separate "use redis" property since we now need to be able to connect to redis via "spring.redis.host" in addition to "spring.redis.sentinel.master" or, possibly in the future, "spring.redis.cluster.nodes", etc